webapp/src/pages/ResponseList/components/ResponseListForm.js

```diff
@@ -176,11 +189,12 @@ class ResponseListForm extends Component {
                 rows.map(val => {
                     let newColumns = Object.keys(val)
                     newColumns.forEach(val => {
-                        tableColumns.includes(val) ? console.log("item already exists") : tableColumns.push(val)
+                        if (!tableColumns.includes(val)) {
+                            tableColumns.push(val)
```

Comment:
You're missing semicolons in a few places - please check

Reply:
I added where missing I think this looks good now